src/shared/components/palette/case-link/components/link-cases/link-cases.component.ts

```diff
@@ -143,11 +158,18 @@ export class LinkCasesComponent implements OnInit {
             caseReference: caseView.case_id,
             reasons: this.getSelectedCaseReasons(),
             createdDateTime: new Date().toISOString(),
-            caseType: caseView.case_type.name,
+            caseType: caseView.case_type.id,
             caseState: caseView.state.name,
             caseService: caseView.case_type.jurisdiction.name,
             caseName: caseView.metadataFields && caseView.metadataFields['caseNameHmctsInternal'] ||  'Case name missing',
           };
+          const ccdApiCaseLinkData = {
+            CaseReference: caseView.case_id,
+            CaseType: caseView.case_type.id,
+            CreatedDateTime: new Date().toISOString(),
+            ReasonForLink: this.getSelectedCCDTypeCaseReason()
+          }
```

Comment:
could this cast to a type model? i.e.
ccdApiCaseLinkData: ModelType = {
            CaseReference: caseView.case_id,
            CaseType: caseView.case_type.id,
            CreatedDateTime: new Date().toISOString(),
            ReasonForLink: this.getSelectedCCDTypeCaseReason()
          }